Aspire Budgeting/Facilities/Images.swift

```diff
@@ -11,4 +11,18 @@ extension Image {
   static let magnifyingGlass = Image(systemName: "magnifyingglass")
   static let multiplyCircleFill = Image(systemName: "multiply.circle.fill")
   static let sheetsIcon = Image("sheetsIcon")
+
+  static let dollarSignCircle = Image(systemName: "dollarsign.circle")
+  static let yenSignCircle = Image(systemName: "yensign.circle")
+  static let sterlingSignCircle = Image(systemName: "sterlingsign.circle")
+  static let rubleSignCircle = Image(systemName: "rublesign.circle")
+  static let euroSignCircle = Image(systemName: "eurosign.circle")
+  static let indianRupeeSignCircle = Image(systemName: "indianrupeesign.circle")
+
+  static let currencySymbols: [Image] = [.dollarSignCircle,
+                                         .yenSignCircle,
+                                         sterlingSignCircle,
+                                         rubleSignCircle,
+                                         euroSignCircle,
+                                indianRupeeSignCircle, ]
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/9fa5987dc8d0836840f2091074e9639f1ee72ccb/Aspire Budgeting/Facilities/Images.swift#L27"></span>All elements in a collection literal should be vertically aligned<br />
<code>collection_alignment</code> <code>Images.swift:27</code></td>
    </tr>
  </tbody>
</table>